examples/blinky/Makefile

```diff
@@ -12,6 +12,8 @@ RIOTBASE ?= $(CURDIR)/../..
 # development process:
 DEVELHELP ?= 1
 
+CFLAGS += -DUART_TXBUF_SIZE=4
```

Comment:
This does not belong in the example :wink:

Reply:
I should probably add an example and/or test that checks whether the transmitting task is really yielding.